Makefile

```diff
@@ -92,10 +94,18 @@ install-identity-scripts: install-datadir
 	install -m 755 -d $(prefix)$(datadir)/mender/identity
 	install -m 755 $(IDENTITY_SCRIPTS) $(prefix)$(datadir)/mender/identity/
 
-install-inventory-scripts: install-datadir
+install-inventory-scripts: install-datadir install-inventory-network-scripts
+	install -m 755 -d $(prefix)$(datadir)/mender/inventory
+	install -m 755 $(INVENTORY_SCRIPTS) $(prefix)$(datadir)/mender/inventory/
+
+install-local-inventory-scripts: install-datadir
```

Comment:
For consistency, I suggest:
```
install-local-inventory-scripts
install-network-inventory-scripts
```
or
 ```
install-inventory-local-scripts
install-inventory-network-scripts
```

Reply:
Yes, true. Let's unify this. I would vote for the second option ?